backend/drm/backend.c

```diff
@@ -51,7 +51,11 @@ static void backend_destroy(struct wlr_backend *backend) {
 static struct wlr_renderer *backend_get_renderer(
 		struct wlr_backend *backend) {
 	struct wlr_drm_backend *drm = (struct wlr_drm_backend *)backend;
-	return drm->renderer.wlr_rend;
+
+	if (drm->parent)
```

Comment:
Style: we always use brackets.

Returning the parent's renderer is the Right Thing To Do™.